students/slupek_bartosz/lesson_03_functions/fibonacci_numbers.py

```diff
@@ -0,0 +1,11 @@
+def fib(n):
+    if n < 0:
+        print("Error n cannot be less than 0")
```

Comment:
lepszym rozwiązaniem było by zwrócenie wyjątku: `raise ValueError`, ponieważ nie jest to typowa sytuacja, oraz przez brak `return` bedziesz zwracal w tym kroku return.
Jeżeli deklarujesz w funkcji, że cos zwracasz poprzez reutrn, to zdefiniuj wszystkie wyjscia z funkcji, a dla sytuacji nie typowych uzyj wyjątków.